tests/unit/parsers/test.manifestjson.js

```diff
@@ -2869,7 +2900,50 @@ describe('ManifestJSONParser', () => {
         linter.collector,
         {
           io: { files: { 'background_worker.js': '' } },
-          schemaValidatorOptions: { maxManifestVersion: 3 },
+          schemaValidatorOptions: {
+            maxManifestVersion: 3,
+            mv3BackgroundServiceWorker: true,
+          },
+        }
+      );
+
+      expect(manifestJSONParser.isValid).toBeTruthy();
+    });
+
+    it('does not error if background.page is used with manifest_version: 3', () => {
+      const linter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({
+        manifest_version: 3,
+        background: { page: 'background_page.html' },
```

Comment:
What if we add extra properties?

Reply:
from what I recall extra property wouldn't be allowed by the manifest schema, both unknown ones and persistent which was turned off by default, if you think it would be worth it I can also add test coverage for that if some isn't already provided by some other tests (I haven't checked to be honest).